index.js

```diff
@@ -20,6 +22,7 @@ function RFXComPlatform(log, config, api) {
   this.config = config || { platform: 'RFXCom' }
   this.tty = this.config.tty || '/dev/ttyUSB0'
   this.debug = this.config.debug || false
+  this.exposeApi = this.config.exposeApi || true
```

Comment:
Please default this to false. Alternatively, you could drop `exposeApi` and just see if `apiPort` has been set to a non-zero number.